apps/asap-server/src/controllers/research-outputs.ts

```diff
@@ -136,20 +136,24 @@ export default class ResearchOutputs implements ResearchOutputController {
     };
   }
   async create({
-    teamId,
+    teams,
     ...researchOutputData
   }: ResearchOutputInputData): Promise<Partial<ResearchOutputResponse>> {
     const { id: researchOutputId } = await this.createResearchOutput(
       researchOutputData,
     );
-    await this.associateResearchOutputToTeam(teamId, researchOutputId);
+    await Promise.all(
```

Comment:
What about retries?
Could we add here code to retry this assignment on failure?

Reply:
I'm not sure there's much we can do here. The RO needs to be associated with a team, and this much occurs after the RO is created. 

In terms of retrying, how long do we retry? How many times? The user is waiting for this to return before they can continue, should we take the create event asynchronous and return straight away? What happens if the create event continues to fail, should we DLQ?

There are no transactions in Squidex, (as far as I know) so we can't simply rollback.

We could "manually" rollback, i.e. delete the RO we've just created. But what happens if that delete fails? Do we have to retry, for how long? Do we DLQ or put the event somewhere?